Fix sortstart event to fire everytime

mfriesen opened the pull request:
The sortstart event only fired if the lists were connected or the same, but won't fire across 2 different lists or if they are not connected. see #296

mfriesen committed:
fixed sortstart/sortstop/sortupdate events to fire with acceptFrom

lukasoppermann commented on src/html.sortable.js:
```diff
@@ -610,7 +618,7 @@ var sortable = function (sortableElements, options) {
       startList = _serialize(startParent)
       // dispatch sortstart event on each element in group
-      _dispatchEventOnConnected(sortableElement, _makeEvent('sortstart', {
+      _dispatchEvent(sortableElement, _makeEvent('sortstart', {
```
Sorry, but looking at it now, couldn't we just do `sortableElement.dispatchEvent( _makeEvent('sortstart', {…` instead if using the `_dispatchEvent` function? Sorry, I did not catch that before you wrote it.

Also the comment should be updated

mfriesen commented:
@lukasoppermann I've fixed it and I've changed sortupdate & sortstop to match. I don't believe this will break backwards compatibility

lukasoppermann commented:
Hey @mfriesen did you rebase to master? Is it possible that you forgot to push the fixes? Because it looks the same to me.

mfriesen commented:
@lukasoppermann my bad. It's up now.

lukasoppermann commented:
Hey, the question with removing them is, will `sortupdate` & `sortstop` be called on the not connected sortables if dropped on them? Did you check this? Could you add tests for this as well? Basically the same tests with not connected, which should turn out to be false on the called variablesß

Thank you.

mfriesen commented:
@lukasoppermann ok, I think I fixed it up much better. The real problem is that the events don't fire when acceptFrom is used.
You'll also noticed I removed firing of the 'dropend' event from the 'drop' event. The browser does this already, so this didn't actual do anything.

Commit:
Merge branch 'master' into sortstart

lukasoppermann commented:
Thank you, looks good! :+1:
